### PR TITLE
Ds 37 firebase setup

### LINKED ISSUE #37: Firebase Setup
**Is your feature request related to a problem? Please describe.**
The web application created using React.js needs to be seen by the world.

**Describe the solution you'd like**
A solution would be to use Firebase as a hosting platform, and enable continuous deployment between Firebase and GitHub. This would require on the possibility of creating some setup files like .yml, .json, etc.

**Describe alternatives you've considered**
If continuous deployment fails, then the alternative would be not implement it. If Firebase fails, then the alternative would be to use Netlify.

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
1. Setup firebase for web deployment

Closes #37

### CODE REVIEW

File: .github/main.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        uses: actions/download-artifact@v2
+        with:
+          name: build
+          path: build
+      - name: Deploy to Firebase
+        uses: w9jds/firebase-action@master
+        with:
+          args: deploy --only hosting
+        env:
+          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
```
Comment (BlackCubes) [inline .github/main.yml:43]: Here we go.....

**Review by JorgeLVilla — APPROVED** [review]
Great Job! Looks good to me.

**Review by BlackCubes — APPROVED** [review]
It's time for continuous deployment 🎂

**Review by JayDampitan — APPROVED** [review]
Great job ya'll. Thanks for all the help and good luck!

**Pull request merged**